scripts/versions

```diff
@@ -3,7 +3,7 @@ set -euo pipefail
 
 # Versions to be used
 declare -A VERSIONS=(
-    ["go"]=1.15.3
+    ["go"]=1.15.4
```

Comment:
going along with what @saschagrunert said above, after adding this, one needs to run `make test-images` and push the images to quay and update the image locations in the `stdenv` section of .circleci/config.yml

Reply:
Alas, I don't know much about it... Can you please do that?

I mean, update from 1.15.3 from 1.15.4 is probably not that important; but 1.14 to 1.15 might be.

Reply:
done